tests/urlpatterns_reverse/tests.py

```diff
@@ -225,6 +229,18 @@ def test_patterns_reported(self):
             # exception
             self.fail("Expected a NoReverseMatch, but none occurred.")
 
+    def test_script_name_escaping(self):
+        from django.core.urlresolvers import get_script_prefix, set_script_prefix
+        old_prefix = get_script_prefix()
```

Comment:
Since other tests use the prefix-kwarg of reverse directly, this seems not necessary. But since the prefix-kwarg is only used for testing, it might be better to make a context-manager of set_script_prefix. remove the non-documented prefix-kwarg and update the tests.